nix/yazi-unwrapped.nix

```diff
@@ -1,67 +1,86 @@
 {
-  makeRustPlatform,
-  rustToolchain,
+  rustPlatform,
   version ? "git",
-  rev,
-  date,
+  rev ? "unknown",
+  date ? "19700101",
   lib,
 
   installShellFiles,
   stdenv,
   darwin,
+  rust-jemalloc-sys,
 
   imagemagick,
 }:
+let
+  src = lib.fileset.toSource {
+    root = ../.;
+    fileset = lib.fileset.intersection (lib.fileset.fromSource (lib.sources.cleanSource ../.)) (
+      lib.fileset.unions [
+        ../assets
+        ../yazi-adapter
```

Comment:
Could we add all `yazi-*`?

https://github.com/sxyazi/yazi/blob/7a380c2f0fdaec87992836e21dc8b6cd251af980/Cargo.toml#L3